packages/react/src/components/Forms/Select/mono-common.tsx

```diff
@@ -175,7 +175,7 @@ export const SelectMonoAux = ({
                           downshiftReturn.selectedItem === item,
                         "c__select__menu__item--disabled": item.disabled,
                       })}
-                      key={`${item.value}${index.toString()}`}
+                      key={`${optionToValue(item)}${index}`}
```

Comment:
To remove the eslint warning:
```suggestion
                      key={`${optionToValue(item)}${index.toString()}`}
```

Same with the others.